src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.ts

```diff
@@ -26,11 +27,19 @@ interface DotEditPageNavItem {
 })
 export class DotEditPageNavComponent implements OnChanges {
     @Input() pageState: DotRenderedPageState;
+    @Output() buttonClicked: EventEmitter<{ inode: string; label: string }> = new EventEmitter();
```

Comment:
I think we should call this like `action`, @freddyucv @hmoreras thoughts?

Reply:
action is good for me